.github/workflows/vib.yaml

```diff
@@ -54,9 +65,148 @@ jobs:
           secrets: |-
             package-carto-values:projects/carto-tnt-onp-${{ env.ONPREM_ID }}/secrets/selfhosted-k8s-customer-package
             package-carto-secrets:projects/carto-tnt-onp-${{ env.ONPREM_ID }}/secrets/selfhosted-k8s-customer-package-secrets
+            e2e-secrets:projects/carto-terraform-ci-cd/secrets/github-carto3-e2e-secrets-${{ env.ENVIRONMENT }}
           # yamllint enable
       - name: Set up yq
         uses: frenck/action-setup-yq@v1
+      - name: Retrieve chart info
+        id: info
+        shell: bash
+        run: |
+          set -ex
+          echo "::set-output name=carto_version::$(yq .appVersion chart/Chart.yaml)"
+          echo "::set-output name=custom_run_id::${{ github.repository }}:${{ github.run_id }}[${{ github.run_attempt }}](${{ matrix.target-platform }})"
+      - name: Clone cloud-native
+        uses: actions/checkout@v3
+        with:
+          ref: '${{ steps.info.outputs.carto_version }}'
+          # ref: 'vib-cypress'
+          # ref: 'beltran-rubo-vib-cypress'
+          repository: 'CartoDB/cloud-native'
+          # repository: 'beltran-rubo/cloud-native'
+          token: '${{ secrets.X_GITHUB_CARTOFANTE }}'
+          path: '${{ env.CARTO_CLONE_PATH }}'
+          persist-credentials: 'false'
+      - name: Package E2E tests from cloud-native
+        id: e2e-package
+        shell: bash
+        env:
+          E2E_SECRETS: ${{ steps.secrets.outputs.e2e-secrets }}
+        run: |
+          # shellcheck disable=SC2002,SC2129
+          set -ex
+
+          ### Basic Info
+          du -sh "${CARTO_CLONE_PATH}/workspace-www"
+          cd "${CARTO_CLONE_PATH}/workspace-www"
+          git status
+
+          ### Edit workspace-www folder
+
+          #   ## HotFix until we go against main
+          # cat .env | head -n 8 > .env.edited
+          # mv -f .env.edited .env
+          # jq 'del(.projectId)' cypress.json > cypress.json.edited
+          # # jq '.projectId = "ggohpq"' cypress.json > cypress.json.edited
+          # mv -f cypress.json.edited cypress.json
+          # #   ## BASEURL WITH PROTOCOL !
+          # # jq '.baseUrl = "https://carto.vmw"' cypress.json > cypress.json.edited
```

Comment:
We should move from `baseURL` option into the `cypress.json` to CYPRESS_BASE_URL env var to avoid [this Cypress issue](https://github.com/cypress-io/cypress/issues/20647)

Reply:
Ensuring `CYPRESS_BASE_URL` is in `.env` file